lib/virtual_device.js

```diff
@@ -121,6 +121,27 @@ VirtualDevice.prototype.call = function(/* transition, args, cb */) {
 
 };
 
+VirtualDevice.prototype.available = function(transition) {
+  return !!this._getAction(transition);
+};
+
+VirtualDevice.prototype.transitionsAvailable = function() {
```

Comment:
Oh, I see on `zetta-device`, it gets used for sending log stream messages.  This should probably be private in both cases.  Do we really need this for VirtualDevice?